### PR TITLE
Lidar motion

### PR DESCRIPTION
**Pull request by LauraSManns**

### CODE REVIEW

File: camille/core.cpp [hunk 1]
```diff
@@ -11,14 +11,30 @@ namespace py = pybind11;
 static const auto dense = py::array::c_style | py::array::forcecast;
 
 namespace {
+
+namespace vec3 {
+    using vec3 = std::tuple<double, double, double>;
+    double x(vec3 v) {
+        return std::get<0>(v);
+    }
+}
```
Comment (JensGM) [inline camille/core.cpp:20]:
```suggestion
struct rotation {
    double pitch;
     ...
};
```

File: camille/core.cpp [hunk 2]
```diff
@@ -322,14 +506,13 @@ windfield_desc calc_windfield_desc(
     wf_desc.veer = std::nan("");
     wf_desc.upper = std::move(upper_desc);
     wf_desc.lower = std::move(lower_desc);
-
+
```
Comment (mrSundvoll) [inline camille/core.cpp:509]: Remove extra whitespaces

File: camille/core.cpp [hunk 3]
```diff
@@ -338,7 +521,7 @@ T* checked_ptr(py::array_t<T, dense>& in,
                const ssize_t expected_ndim,
                const std::vector<ssize_t>& expected_shape) {
     const auto desc = in.request();
-
+
```
Comment (mrSundvoll) [inline camille/core.cpp:524]: Remove extra whitespaces

Comment (LauraSManns) [reply camille/core.cpp]: Done :)

**Review by JensGM — APPROVED** [review]

**Pull request merged**